frontend/src/data/config/MetricConfig.ts

```diff
@@ -333,11 +359,8 @@ export const METRIC_CONFIG: Record<DropdownVarId, VariableConfig[]> = {
         pct_share: {
           metricId: "covid_cases_share",
           fullCardTitleName: "Share of total COVID-19 cases",
-          chartTitle: "Share of total COVID-19 cases with unknown",
```

Comment:
Removing the _chartTitle_  prop will cause the chart title on the Unknown maps to be partially blank.